lms/djangoapps/ccx/utils.py

```diff
@@ -260,3 +261,83 @@ def is_email(identifier):
     except ValidationError:
         return False
     return True
+
+
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Added staff role on ccx to all the staff members of master course.
+
+    Arguments:
+        course (CourseDescriptorWithMixins): Master course instance
+        ccx_key (CCXLocator): CCX course key
+        display_name (str): ccx display name for email
+    """
+    list_staff = list_with_level(course, 'staff')
+    list_instructor = list_with_level(course, 'instructor')
+
+    with ccx_course(ccx_key) as course:
```

Comment:
Perhaps this pattern is already spread throughout the CCX code - but this code is confusing to someone not familiar with CCX. What's the difference between the `course` parameter passed-in and the `course` that's retrieved via the context manager? If they are truly different, is there a different naming convention that could be used here?